lib/PuppeteerSharp.Tests/PageTests/PdfTests.cs

```diff
@@ -1,138 +0,0 @@
-﻿using System;
-using System.IO;
-using System.Threading.Tasks;
-using Xunit;
-using PdfSharp.Pdf.IO;
-using PuppeteerSharp.Media;
-using Xunit.Abstractions;
-
-namespace PuppeteerSharp.Tests.PageTests
-{
-    [Collection("PuppeteerLoaderFixture collection")]
-    public class PdfTests : PuppeteerPageBaseTest
-    {
-        public PdfTests(ITestOutputHelper output) : base(output)
-        {
-        }
-
-        [Fact]
-        public async Task ShouldBeAbleToSaveFile()
```

Comment:
We need to leave this test https://github.com/kblok/puppeteer/blob/c430138845a44e20bff88b49843569f119e08d3d/test/page.spec.js#L1363